File: Server/src/com/mytalk/server/logic/shared/modelClient/Call.java
/**
* Filename: Call.java
* Package: com.mytalk.server.logic.shared.modelclient
* Author: Nicolò Toso
* Date: 2013-04-18
*
* Diary:
*
* Version |
Date
| Version | Date       | Developer | Changes
* --------+------------+-----------+------------------
* 0.1	  |	2013-04-18 | NT        | [+] Creazione classe e metodi   
*
* This software is distributed under GNU/GPL 2.0.
*
* Software licensed to:
* - Zucchetti SRL
*/

package com.mytalk.server.logic.shared.modelClient;

public class Call{
	
	private String speaker; //id dell'user
	private int byteSent;
	private int byteReceived;
	private int duration;
	private String startdate;
	private boolean caller;
	
	public Call(){}
	
	public Call(String s, int bs, int br, int d, String sd, boolean c){
		speaker=s;
		byteSent=bs;
		byteReceived=br;
		duration=d;
		startdate=sd;
		caller=c;
	}
	
	public String getSpeaker(){
		return speaker;
	}
	public void setSpeaker(String n){
		speaker=n;
	}
	public boolean getCaller(){
		return caller;
	}
	public void setCaller(boolean s){
		caller=s;
	}
	public int getDuration(){
		return duration;
	}
	public void setDuration(int t){
		duration=t;
	}
	public int getByteSent(){
		return byteSent;
	}
	public void setByteSent(int n){
		byteSent=n;
	}
	public int getByteReceived(){
		return byteReceived;
	}
	public void setByteReceived(int n){
		byteReceived=n;
	}
	public String getStartdate(){
		return startdate;
	}
	public void setStartDate(String t){
		startdate=t;
	}
	
}

File: Client/com/mytalk/model/modelstruct/Call.js
/**
* Filename: Call.js
* Package: com.mytalk.model.modelstruct
* Dependencies: com.mytalk.model.modelstruct.User
* Author: Mattia Agostinetto
* Date: 2013-04-20
*
* Diary:
*
| Version | Date       | Developer | Changes
* --------+------------+-----------+------------------
* 0.1     | 2013-04-20 | MA        | [+] Scrittura classe del Model
*
* This software is distributed under GNU/GPL 2.0.
*
* Software licensed to:
* - Zucchetti SRL
*/

MyTalk.Call = DS.Model.extend({
  speaker: DS.belongsTo('MyTalk.User'),
  caller: DS.attr('boolean'),
  startDate: DS.attr('date'),
  duration: DS.attr('number'),
  byteSent: DS.attr('number'),
  byteReceived: DS.attr('number')
});